lametro/templates/lametro/legislation.html

```diff
@@ -98,20 +97,25 @@ <h3 class="no-pad-bottom"><i class='fa fa-fw fa-list-ul'></i> History</h3>
                             </tr>
                         </thead>
                         <tbody>
-                            {% for action in actions %}
+                            {% for event in events %}
                                 <tr>
                                     <td class='nowrap text-muted small'>
-                                        <span datetime='{{action.date_dt | format_date_sort }}'>{{action.date_dt|date:'n/d/y'}}</span>
+                                        <span datetime='{{event.date}}'>{{event.date}}</span>
                                     </td>
                                     <td class="small">
-                                        <span class='text-{{action.label}}'>{{action.description | remove_action_subj}}</span>
+                                        <span class='text-default'>{{event.description}}</span>
                                         {% if action.related_organization %}
```

Comment:
Looks like you need to update this section to use your dictionary, as well -- it's operating as if we are still dealing with `Action` objects.